src/vs/workbench/contrib/bulkEdit/browser/bulkEditPreview.ts

```diff
@@ -260,6 +260,17 @@ export class BulkFileOperations {
 			}
 		}
 
+		// sort (once) categories atop which have unconfirmed edits
+		this.categories.sort((a, b) => {
```

Comment:
Should we switch to use a stable sort such as mergeSort from `array.ts`? (not sure if that would have fixed the original problem or not)

Reply:
I think by now all sort-implementations are stable (by spec update and implementation) and that can get rid of mergeSort